tests/test_basic.py

```diff
@@ -96,6 +96,9 @@ def test_fallback_list():
     assert isinstance(params['algorithms'][0], basic.Memorized)
     assert isinstance(params['algorithms'][1], basic.Bias)
 
+def test_fallback_string():
+    algo = basic.Fallback([basic.Memorized(simple_df), basic.Bias()])
+    str(algo)
```

Comment:
This test doesn't have any assertions - two ways I like to fix that:

- if the purpose is just to make sure that it doesn't fail, then add a comment after this line, e.g. `# str succesful, that's what we need`
- we can make the test a little more advanced - e.g. test that the string begins with `Fallback`.